addons/pos_mercado_pago/controllers/main.py

```diff
@@ -0,0 +1,48 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import json
+import logging
+
+from odoo import http
+from odoo.http import request
+
+_logger = logging.getLogger(__name__)
+
+class Webhook(http.Controller):
+    @http.route('/pos_mercado_pago/notification', methods=['POST'], type="http", auth="none", csrf=False)
```

Comment:
You return a JSON please user type="json" and remove csrf=False, since isn't necessary with json route.

See [doc](https://github.com/odoo/enterprise/wiki/security#PY022)

Reply:
I did this but it doesn't work
Mercado Pago send a http POST:
[https://www.mercadopago.com.mx/developers/es/docs/your-integrations/notifications/webhooks](https://www.mercadopago.com.mx/developers/es/docs/your-integrations/notifications/webhooks)